### PR TITLE
Update classes-and-objects.md

### PR DESCRIPTION
**Pull request by badersur**

### CODE REVIEW

**Comment by qinezh** [comment]
Open Publishing Build Service: The pull request content has been published and here are some changed files links of this time:
- Docset 1 - **core-docs**  
  [https://review.docs.microsoft.com/en-us/dotnet/articles/csharp/tour-of-csharp/classes-and-objects?branch=pr-en-us-1078](https://review.docs.microsoft.com/en-us/dotnet/articles/csharp/tour-of-csharp/classes-and-objects?branch=pr-en-us-1078)  

**Status**: SucceededWithWarning. **Message**: Warning occurred: Unable to resolve cross-reference "<a href="xref:Microsoft.Win32.SafeHandles.SafeNCryptKeyHandle" sourcefile="docs/standard/garbagecollection/implementing-dispose.md" sourcestartlinenumber="413" sourceendlinenumber="413">SafeNCryptKeyHandle</a>", "<a href="xref:Microsoft.Win32.SafeHandles.SafeNCryptProviderHandle" sourcefile="docs/standard/garbagecollection/implementing-dispose.md" sourcestartlinenumber="413" sourceendlinenumber="413">SafeNCryptProviderHandle</a>", "<a href="xref:Microsoft.Win32.SafeHandles.SafeNCryptSecretHandle" sourcefile="docs/standard/garbagecollection/implementing-dispose.md" sourcestartlinenumber="413" sourceendlinenumber="413">SafeNCryptSecretHandle</a>"..  
[Open Publishing Report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C9%5C26%5C2a032abb-a76c-bfc8-4231-71ab612989a2%5CPullRequest%5C201609261106303342-1078%5Cworkflow_report.html).

**Review by mairaw — APPROVED** [review]
The changes look great! Thanks @badersur.

File: docs/csharp/tour-of-csharp/classes-and-objects.md [hunk 1]
```diff
@@ -13,15 +13,15 @@ ms.assetid: 63a89bde-0f05-4bc4-b0cd-4f693854f0cd
 ---
 # Classes and objects
 
-***Classes*** are the most fundamental of C#’s types. A class is a data structure that combines state (fields) and actions (methods and other function members) in a single unit. A class provides a definition for dynamically created ***instances*** of the class, also known as ***objects***. Classes support ***inheritance*** and ***polymorphism***, mechanisms whereby ***derived classes*** can extend and specialize ***base classes***.
+*Classes* are the most fundamental of C#’s types. A class is a data structure that combines state (fields) and actions (methods and other function members) in a single unit. A class provides a definition for dynamically created *instances* of the class, also known as *objects*. Classes support *inheritance* and *polymorphism*, mechanisms whereby *derived classes* can extend and specialize *base classes*.
```
Comment (stevehoag) [inline docs/csharp/tour-of-csharp/classes-and-objects.md]: Should be a lowercase l in *poLymorphism*

File: docs/csharp/tour-of-csharp/classes-and-objects.md [hunk 2]
```diff
@@ -75,12 +75,12 @@ A class definition may specify a set of type parameters by following the class n
 
 [!code-csharp[Pair](../../../samples/snippets/csharp/tour/classes-and-objects/Pair.cs#L3-L7)]
 
-A class type that is declared to take type parameters is called a ***generic class type***. Struct, interface and delegate types can also be generic.
+A class type that is declared to take type parameters is called a *generic class type*. Struct, interface and delegate types can also be generic.
```
Comment (stevehoag) [inline docs/csharp/tour-of-csharp/classes-and-objects.md]: Lowercase l in *generic cLass type*

File: docs/csharp/tour-of-csharp/classes-and-objects.md [hunk 3]
```diff
@@ -106,35 +106,35 @@ In the following example, each instance of the `Color` class has a separate copy
 
 [!code-csharp[ColorClass](../../../samples/snippets/csharp/tour/classes-and-objects/Color.cs#L3-L17)]
 
-As shown in the previous example, ***read-only fields*** may be declared with a `readonly` modifier. Assignment to a `readonly` field can only occur as part of the field’s declaration or in a constructor in the same class.
+As shown in the previous example, *read-only fields* may be declared with a `readonly` modifier. Assignment to a `readonly` field can only occur as part of the field’s declaration or in a constructor in the same class.
```
Comment (stevehoag) [inline docs/csharp/tour-of-csharp/classes-and-objects.md]: Lowercase l in *read-only fields*

File: docs/csharp/tour-of-csharp/classes-and-objects.md [hunk 4]
```diff
@@ -106,35 +106,35 @@ In the following example, each instance of the `Color` class has a separate copy
...
 
-Parameters are used to pass values or variable references to methods. The parameters of a method get their actual values from the ***arguments*** that are specified when the method is invoked. There are four kinds of parameters: value parameters, reference parameters, output parameters, and parameter arrays.
+Parameters are used to pass values or variable references to methods. The parameters of a method get their actual values from the *arguments* that are specified when the method is invoked. There are four kinds of parameters: value parameters, reference parameters, output parameters, and parameter arrays.
 
-A ***value parameter*** is used for input parameter passing. A value parameter corresponds to a local variable that gets its initial value from the argument that was passed for the parameter. Modifications to a value parameter do not affect the argument that was passed for the parameter. 
+A *value parameter* is used for input parameter passing. A value parameter corresponds to a local variable that gets its initial value from the argument that was passed for the parameter. Modifications to a value parameter do not affect the argument that was passed for the parameter.
```
Comment (stevehoag) [inline docs/csharp/tour-of-csharp/classes-and-objects.md]: Lowercase l

**Comment by badersur** [comment]
You are welcome @mairaw. I've been confused by formatting so I didn't change a lot! There may be others to be fixed :(

File: docs/csharp/tour-of-csharp/classes-and-objects.md [hunk 5]
```diff
@@ -150,19 +150,19 @@ is equivalent to writing the following.
 
 A method’s body specifies the statements to execute when the method is invoked.
 
-A method body can declare variables that are specific to the invocation of the method. Such variables are called ***local variables***. A local variable declaration specifies a type name, a variable name, and possibly an initial value. The following example declares a local variable `i` with an initial value of zero and a local variable `j` with no initial value.
+A method body can declare variables that are specific to the invocation of the method. Such variables are called *local variables*. A local variable declaration specifies a type name, a variable name, and possibly an initial value. The following example declares a local variable `i` with an initial value of zero and a local variable `j` with no initial value.
```
Comment (stevehoag) [inline docs/csharp/tour-of-csharp/classes-and-objects.md]: Lowercase l

File: docs/csharp/tour-of-csharp/classes-and-objects.md [hunk 6]
```diff
@@ -150,19 +150,19 @@ is equivalent to writing the following.
...
 
 [!code-csharp[Squares](../../../samples/snippets/csharp/tour/classes-and-objects/Squares.cs#L3-L17)]
 
-C# requires a local variable to be ***definitely assigned*** before its value can be obtained. For example, if the declaration of the previous `i` did not include an initial value, the compiler would report an error for the subsequent usages of `i` because `i` would not be definitely assigned at those points in the program.
+C# requires a local variable to be *definitely assigned* before its value can be obtained. For example, if the declaration of the previous `i` did not include an initial value, the compiler would report an error for the subsequent usages of `i` because `i` would not be definitely assigned at those points in the program.
```
Comment (stevehoag) [inline docs/csharp/tour-of-csharp/classes-and-objects.md]: Lowercase l

File: docs/csharp/tour-of-csharp/classes-and-objects.md [hunk 7]
```diff
@@ -180,13 +180,13 @@ Note that the `SetNextSerialNo` and `GetNextSerialNo` static methods are invoked
 
 ### Virtual, override, and abstract methods
 
-When an instance method declaration includes a `virtual` modifier, the method is said to be a ***virtual method***. When no virtual modifier is present, the method is said to be a ***nonvirtual method***.
+When an instance method declaration includes a `virtual` modifier, the method is said to be a *virtual method*. When no virtual modifier is present, the method is said to be a *nonvirtual method*.
```
Comment (stevehoag) [inline docs/csharp/tour-of-csharp/classes-and-objects.md]: Lowercase l

File: docs/csharp/tour-of-csharp/classes-and-objects.md [hunk 8]
```diff
@@ -180,13 +180,13 @@ Note that the `SetNextSerialNo` and `GetNextSerialNo` static methods are invoked
...
 
-When a virtual method is invoked, the ***run-time type*** of the instance for which that invocation takes place determines the actual method implementation to invoke. In a nonvirtual method invocation, the ***compile-time type*** of the instance is the determining factor.
+When a virtual method is invoked, the *run-time type* of the instance for which that invocation takes place determines the actual method implementation to invoke. In a nonvirtual method invocation, the *compile-time type* of the instance is the determining factor.
```
Comment (stevehoag) [inline docs/csharp/tour-of-csharp/classes-and-objects.md]: Lowercase l

File: docs/csharp/tour-of-csharp/classes-and-objects.md [hunk 9]
```diff
@@ -206,23 +206,23 @@ The following program uses the `Expression` classes to evaluate the expression `
 
 ### Method overloading
 
-Method ***overloading*** permits multiple methods in the same class to have the same name as long as they have unique signatures. When compiling an invocation of an overloaded method, the compiler uses ***overload resolution*** to determine the specific method to invoke. Overload resolution finds the one method that best matches the arguments or reports an error if no single best match can be found. The following example shows overload resolution in effect. The comment for each invocation in the `Main` method shows which method is actually invoked.
+Method *overloading* permits multiple methods in the same class to have the same name as long as they have unique signatures. When compiling an invocation of an overloaded method, the compiler uses *overload resolution* to determine the specific method to invoke. Overload resolution finds the one method that best matches the arguments or reports an error if no single best match can be found. The following example shows overload resolution in effect. The comment for each invocation in the `Main` method shows which method is actually invoked.
```
Comment (stevehoag) [inline docs/csharp/tour-of-csharp/classes-and-objects.md]: Lowercase l

File: docs/csharp/tour-of-csharp/classes-and-objects.md [hunk 10]
```diff
@@ -234,9 +234,9 @@ Unlike other members, instance constructors are not inherited, and a class has n
...
+*Properties* are a natural extension of fields. Both are named members with associated types, and the syntax for accessing fields and properties is the same. However, unlike fields, properties do not denote storage locations. Instead, properties have *accessors* that specify the statements to be executed when their values are read or written.
 
-A property is declared like a field, except that the declaration ends with a get accessor and/or a set accessor written between the delimiters `{` and `}` instead of ending in a semicolon. A property that has both a get accessor and a set accessor is a ***read-write property***, a property that has only a get accessor is a ***read-only property***, and a property that has only a set accessor is a ***write-only property***.
+A property is declared like a field, except that the declaration ends with a get accessor and/or a set accessor written between the delimiters `{` and `}` instead of ending in a semicolon. A property that has both a get accessor and a set accessor is a *read-write property*, a property that has only a get accessor is a *read-only property*, and a property that has only a set accessor is a *write-only property*.
```
Comment (stevehoag) [inline docs/csharp/tour-of-csharp/classes-and-objects.md]: Lowercase l in two places

File: docs/csharp/tour-of-csharp/classes-and-objects.md [hunk 11]
```diff
@@ -262,21 +262,21 @@ Indexers can be overloaded, meaning that a class can declare multiple indexers a
...
 
 The `List<T>` class declares a single event member called `Changed`, which indicates that a new item has been added to the list. The Changed event is raised by the `OnChanged` virtual method, which first checks whether the event is `null` (meaning that no handlers are present). The notion of raising an event is precisely equivalent to invoking the delegate represented by the event—thus, there are no special language constructs for raising events.
 
-Clients react to events through ***event handlers***. Event handlers are attached using the `+=` operator and removed using the `-=` operator. The following example attaches an event handler to the `Changed` event of a `List<string>`.
+Clients react to events through *event handlers*. Event handlers are attached using the `+=` operator and removed using the `-=` operator. The following example attaches an event handler to the `Changed` event of a `List<string>`.
```
Comment (stevehoag) [inline docs/csharp/tour-of-csharp/classes-and-objects.md]: Lowercase l

File: docs/csharp/tour-of-csharp/classes-and-objects.md [hunk 12]
```diff
@@ -286,7 +286,7 @@ The first `Console.WriteLine` outputs `True` because the two lists contain the s
 
 ### Finalizers
 
-A ***finalizer*** is a member that implements the actions required to finalize an instance of a class. Finalizers cannot have parameters, they cannot have accessibility modifiers, and they cannot be invoked explicitly. The finalizer for an instance is invoked automatically during garbage collection.
+A *finalizer* is a member that implements the actions required to finalize an instance of a class. Finalizers cannot have parameters, they cannot have accessibility modifiers, and they cannot be invoked explicitly. The finalizer for an instance is invoked automatically during garbage collection.
```
Comment (stevehoag) [inline docs/csharp/tour-of-csharp/classes-and-objects.md]: Lowercase l

**Review by stevehoag — COMMENTED** [review]
Thanks for fixing these @badersur 
It seems that you are capitalizing the letter "l" in your changes - these should be lowercase. Other than that, LGTM.

**Pull request merged**